apps/website/src/icons/IconGithub.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+type IconProps = {
+  size?: number;
+  className?: string;
+};
+
+const IconGithub: React.FC<IconProps> = ({ size = 24, className }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={size}
+    height={size}
+    viewBox="0 0 98 96"
+    className={className}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
```

Comment:
```suggestion
      fill="currentColor"
```
To be consistent with other icons, default to using the text color